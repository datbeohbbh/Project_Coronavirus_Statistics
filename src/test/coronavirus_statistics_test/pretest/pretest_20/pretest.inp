36 24
q 1 2 281
q 1 33 196
? 4 2
q 3 2 186
? 4 1
q 2 2 58
? 4 1
? 4 2
? 6
? 6
? 4 2
q 2 33 159
q 2 2 17
? 4 1
q 2 2 18
q 2 33 5
q 1 31 159
q 1 4 494
? 6
? 4 1
? 4 4
? 6
? 7
? 4 3