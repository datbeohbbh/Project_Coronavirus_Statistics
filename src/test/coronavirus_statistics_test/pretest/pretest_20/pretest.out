Query ? 4 2 :
The answer is the city with the index 2 
- Information of the city 2 -
-> Number of current patients : 281
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 2 
- Information of the city 2 -
-> Number of current patients : 95
-> Has recovered : 0
-> Was dead : 186

Query ? 4 1 :
The answer is the city with the index 2 
- Information of the city 2 -
-> Number of current patients : 37
-> Has recovered : 58
-> Was dead : 186

Query ? 4 2 :
The answer is the city with the index 33 
- Information of the city 33 -
-> Number of current patients : 196
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 2
Quantity: 58 people

Query ? 6 :
The city with the highest recovery is city with the index 2
Quantity: 58 people

Query ? 4 2 :
The answer is the city with the index 33 
- Information of the city 33 -
-> Number of current patients : 196
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 2 
- Information of the city 2 -
-> Number of current patients : 20
-> Has recovered : 75
-> Was dead : 186

Query ? 6 :
The city with the highest recovery is city with the index 33
Quantity: 164 people

Query ? 4 1 :
The answer is the city with the index 2 
- Information of the city 2 -
-> Number of current patients : 2
-> Has recovered : 93
-> Was dead : 186

Query ? 4 4 :
The answer is the city with the index 4 
- Information of the city 4 -
-> Number of current patients : 494
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 33
Quantity: 164 people

Query ? 7 :
The city with the highest death is city with the index 2
Quantity: 186 people

Query ? 4 3 :
The answer is the city with the index 31 
- Information of the city 31 -
-> Number of current patients : 159
-> Has recovered : 0
-> Was dead : 0

